packages/edit-post/src/components/header/index.js

```diff
@@ -61,8 +81,23 @@ function Header( { setEntitiesSavedStatesCallback } ) {
 						setEntitiesSavedStatesCallback
 					}
 				/>
-				<PinnedItems.Slot scope="core/edit-post" />
-				<MoreMenu />
+				{ ( isLargeViewport || ! showIconLabels ) && overflowItems }
+				{ showIconLabels && ! isLargeViewport && (
+					<Dropdown
+						contentClassName="edit-post-header__dropdown"
+						position="bottom right"
+						renderToggle={ ( { isOpen, onToggle } ) => (
+							<Button
+								className="button-toggle"
+								aria-expanded={ isOpen }
+								icon={ chevronDown }
+								isSecondary
+								onClick={ onToggle }
+							/>
```

Comment:
Similar again here, this button seems to need a label.

The button styles in the dropdown also look like they need some tweaking:
<img width="96" alt="Screenshot 2020-08-13 at 4 08 04 pm" src="https://user-images.githubusercontent.com/677833/90111086-b7f41300-dd80-11ea-8227-28be38e77e5b.png">